src/crypto/CHIPCryptoPAL.h

```diff
@@ -1523,30 +1538,26 @@ class SymmetricKeyContext
      * for plaintext, and ciphertext.
      * @return CHIP_ERROR
      */
-    virtual CHIP_ERROR DecryptMessage(const ByteSpan & ciphertext, const ByteSpan & aad, const ByteSpan & nonce,
+    virtual CHIP_ERROR MessageDecrypt(const ByteSpan & ciphertext, const ByteSpan & aad, const ByteSpan & nonce,
                                       const ByteSpan & mic, MutableByteSpan & plaintext) const = 0;
 
     /**
      * @brief Perform privacy encoding as described in 4.8.2. (Privacy Processing of Outgoing Messages)
-     * @param[in,out] header    Message header to encrypt
-     * @param[in] session_id    Outgoing SessionID
-     * @param[in] payload       Encrypted payload
-     * @param[in] mic       Outgoing Message Integrity Check
+     * @param[in] input         Message header to privacy encrypt
+     * @param[in] nonce         Privacy Nonce = session_id | mic
+     * @param[out] output       Message header obfuscated
      * @return CHIP_ERROR
      */
-    virtual CHIP_ERROR EncryptPrivacy(MutableByteSpan & header, uint16_t session_id, const ByteSpan & payload,
-                                      const ByteSpan & mic) const = 0;
+    virtual CHIP_ERROR PrivacyObfuscate(const ByteSpan & input, const ByteSpan & nonce, MutableByteSpan & output) const = 0;
```

Comment:
What's wrong with "encrypt" / "decrypt"? I know the spec uses the word "obfuscate" in some paragraphs, but I see no reason to use an obscure synonym, specially when we use "encrypt" / "decrypt" everywhere else.

Reply:
We pivoted to "obfuscate" to provide better clarity on the lack of security guarantees of the privacy header feature, as raised in these two comments:
_Originally posted by @tcarmelveilleux in https://github.com/project-chip/connectedhomeip/pull/21555#discussion_r936967045_
_Originally posted by @bzbarsky-apple in https://github.com/project-chip/connectedhomeip/pull/21555#discussion_r937790096_

Personally, I prefer as you do "PrivacyEncrypt" as that matches the name of the Ch02 crypto primitive in the spec, and the term "privacy" provides enough reference to the feature name to signal associated security caveats.  The preference isn't strong however, and I'm ultimately neutral on which way we go for the naming.

@bzbarsky-apple ?

Reply:
I can live with making these operations' names match their spec names.

Reply:
Ok, renamed to align with Ch02 primitives.